rfcs/rfc-2-font-measuring.md

```diff
@@ -42,13 +31,19 @@ pub struct KayakContext {
 }
 
 impl KayakContext {
-    pub fn get_asset<T>(&self, asset_handle: AssetHandle<T>) -> &T {
-        // Throw error if AssetStorage doesn't exist in the asset resources.
+    // Binding returned here to allow widgets to track loading.
+    pub fn get_asset<T>(&self, asset_handle: String) -> &Binding<Option<T>> {
+        // Create new asset if asset doesn't exist, and sets it to None.
+        ..
+    }
```

Comment:
This will also add the current widget to `binding.when_changed(...)`, right?

Reply:
Yes.